### DISCUSSION TITLE
Using libXNVMe-shared.so

### DISCUSSION THREAD

**Discussion by akash1912** [OpenMPDK/xNVMe#14] [General]
I want to use libXNVMe-shared.so as dynamic library, but in xnvme.io it mentioned that it does not include spdk. Even if in configuration spdk is enabled
libxnvme-shared.so, this is the shared library version of xNVMe, this also does not come with batteries included, so when linking with, or dynamically loading, the shared library version of xNVMe then you also have to link or load the third-party dependencies.

Can Anyone Kindly tell me where I can find the shared so for spdk, or how to build it.

**Comment by karlowich** [comment]
Hi @akash1912 , sorry you didn't get an answer to your question.

I know it is a long time ago, but if it is still relevant, I will happily help out :)
